pkg/controller/jenkins/configuration/base/validate_test.go

```diff
@@ -356,40 +355,13 @@ func TestValidateJenkinsMasterPodEnvs(t *testing.T) {
 		got := baseReconcileLoop.validateJenkinsMasterPodEnvs()
 		assert.Nil(t, got)
 	})
-	t.Run("override JENKINS_HOME env", func(t *testing.T) {
-		jenkins := v1alpha2.Jenkins{
-			Spec: v1alpha2.JenkinsSpec{
-				Master: v1alpha2.JenkinsMaster{
-					Containers: []v1alpha2.Container{
-						{
-							Env: []v1.EnvVar{
-								{
-									Name:  "JENKINS_HOME",
-									Value: "",
-								},
-								{
-									Name:  constants.JavaOpsVariableName,
-									Value: validJenkinsOps,
-								},
-							},
-						},
-					},
-				},
-			},
-		}
-		baseReconcileLoop := New(nil, nil, logf.ZapLogger(false),
-			&jenkins, false, false, nil, nil, nil)
-		got := baseReconcileLoop.validateJenkinsMasterPodEnvs()
-
-		assert.Equal(t, got, []string{"Jenkins Master container env 'JENKINS_HOME' cannot be overridden"})
-	})
```

Comment:
You've deleted the test but not the validation https://github.com/jenkinsci/kubernetes-operator/blob/master/pkg/controller/jenkins/configuration/base/validate.go#L186.